pygrocy/grocy.py

```diff
@@ -434,3 +472,15 @@ def tasks(self) -> List[Task]:
 
     def complete_task(self, task_id):
         return self._api_client.complete_task(task_id)
+
+    def meal_plan(self, get_details: bool = False) -> List[MealPlanItem]:
+        raw_meal_plan = self._api_client.get_meal_plan()
+        meal_plan = [MealPlanItem(data) for data in raw_meal_plan]
+
+        if get_details:
+            for item in meal_plan:
+                item.get_details(self._api_client)
+        return meal_plan
+
+    def recipe(self, recipe_id: int) -> RecipeDetailsResponse:
```

Comment:
Same as with the `recipe` property of `MealPlanItem` please map the response to a `RecipeDetails` object.